app/javascript/components/import/Import.js

```diff
@@ -116,16 +143,22 @@ class Import extends React.Component {
                   aria-valuemax={this.state.phased.length}></div>
               </div>
             )}
-            <h5>Please review the monitorees that are about to be imported below. You can individually accept each monitoree, or accept all at once.</h5>
+            <h5>
+              Please review the monitoree records that are about to be imported. You can individually accept or reject each record below. You can choose to
+              import all unique records or all records (including duplicates) by selecting Accept.
+            </h5>
             <Button
               variant="primary"
               className="btn-lg my-2"
               onClick={() =>
                 this.handleConfirm(
-                  'This will not import already rejected or re-import already accepted monitorees listed below. If duplicates are detected, you will be presented with an option to include them.'
+                  `This will import all records listed below that you did not manually accept or reject.If potential duplicates have been detected, check the box if you would like to import them.`
                 )
               }>
-              Accept All
+              Accept
+            </Button>
+            <Button variant="primary" className="btn-lg my-2 ml-2" onClick={() => this.stopImport()}>
+              Stop Import
```

Comment:
This button should be disabled (or even hidden) if there isn't actually anything to stop. Right now, if I click it, it actually creates a record (and does the next one in line if I keep clicking). Looks like you already created `acceptedAllStarted`, so probably as simple as disabling/conditionally rendering on that.